src/logging/loggerConfigResolver.spec.ts

```diff
@@ -61,5 +63,25 @@ describe('loggerConfigResolver', () => {
         logger.warn('test')
       }).not.toThrow()
     })
+
+    it('redacts logs via provided config', () =>
+      new Promise((done) => {
```

Comment:
shouldn't the promise be either returned or awaited?

Reply:
It's the way vitest does "doneCallback" style tests.

Isn't it returned i.e. I have:
```
() => new Promise()
```
Not:
```
() => { new Promise() }
```

Recommendation from vitest here: https://vitest.dev/guide/migration.html#done-callback

Reply:
you are right, I missed the absence of curly braces